### PR TITLE
tweaking collection permissions so contributor can view

### PR DESCRIPTION
**Pull request by vsoch**

### CODE REVIEW

File: shub/apps/main/models.py [hunk 1]
```diff
@@ -131,6 +135,10 @@ def has_view_permission(instance, request):
     if user in contributors:
         return True
 
+    # Team contributors can view
+    if user.is_team_member(collection):
```
Comment (victorsndvg) [inline shub/apps/main/models.py]: `collection` does not exists in this scope, it's called `instance`

File: shub/apps/main/models.py [hunk 2]
```diff
@@ -84,12 +84,16 @@ def has_edit_permission(instance, request):
...
-    if user.is_staff is True:
+    if user.is_staff:
         return True
 
-    if user.is_superuser is True:
+    if user.is_superuser:
         return True
 
+    # Team owners can edit
+    if user.is_team_owner(collection):
```
Comment (victorsndvg) [inline shub/apps/main/models.py]: collection does not exists in this scope, it's called instance

**Comment by vsoch** [comment]
okay changed collection --> instance

**Comment by vsoch** [comment]
okay, merging.

**Pull request merged**